src/org/labkey/test/tests/issues/IssuesAttachmentTest.java

```diff
@@ -41,7 +42,7 @@
 
 import static org.junit.Assert.assertFalse;
 
-@Category({DailyC.class})
+@Category({Issues.class, DailyC.class})
```

Comment:
This is unnecessary but harmless. The test runner will add this test to the issues suite based on the package name (`org.labkey.test.tests.issues`).

Reply:
Ah cool. I didn't know that. I was using the category annotations as a way to find out what existing automated tests needed to be converted. That is one of the reasons I added it here.